foundry/foundry.py

```diff
@@ -498,3 +512,41 @@ def download_file(file):
             print(results)
 
         return self
+
+    def build(self, spec, globus=False, interval=3, file=False):
+        """Build a Foundry Data Package
```

Comment:
Could you add a little more clarification in the docstring on how this is used? Looks like it instantiates the foundry data package specification but I'm not entirely sure how that would be used.

Reply:
after spending some more time on this function, it looks like it actually downloads the data package per the FoundrySpec? is that right?